boopickle/shared/src/test/scala/java/security/SecureRandom.scala

```diff
@@ -0,0 +1,6 @@
+package java.security
+
+// dummy unsecure SecureRandom implementation for tests
+// for real usage, use an implementation
+// like https://github.com/lolgab/scala-native-crypto
+class SecureRandom extends java.util.Random
```

Comment:
This is odd, do we need to include a class in the java namespace?
Is this only needed for native?

Reply:
Yes, it's needed only on native, I need to move it to the right folder. I put it on `shared` by mistake.
I added it to not have to setup the openssl installation for tests.

Reply:
Actually, it is needed also with Scala.js, so I added it to the `js-native` source directory.